packages/ldes-manager/orm/sequelize.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
```

Comment:
You apparantly also pushed the `.js` files. You only have to commit the `.ts` files

Reply:
Apparently js files outside bin or lib aren't ignored. Fixed that just now !